src/Scene.cpp

```
@@ -35,6 +39,56 @@ void Raytracer::Scene::updateImage() {
+    if (_inputFilePath[i] == '/') {
+      indexCompleteFileName = i;
+      break;
+    }
+  }
+  outputFileName = _inputFilePath.substr(
+      indexCompleteFileName,
+      _inputFilePath.length() - indexCompleteFileName - EXTENSION_LENGHT);
+  for (const auto &entry :
+       std::filesystem::directory_iterator("./screenshots/")) {
+    std::string path = entry.path();
+    if (path.find(outputFileName)) {
```

Cppcheck (2.10-2)

[performance] Inefficient usage of string

(stlIfStrFind)